includes/wikia/api/DesignSystemApiController.class.php

```diff
@@ -195,4 +203,48 @@ private function isWikiaOrgCommunity() {
 
 		return wfGetBaseDomainForHost( parse_url( $wgServer, PHP_URL_HOST ) ) === $wgWikiaOrgBaseDomain;
 	}
+
+
+	/**
+	 * External API request to fandom store
+	 */
+	public function getFandomStoreData() {
+		global $wgCityId, $wgMemc, $wgFandomStoreMap;
+
+		// api for fandom store
+		$uri = 'http://138.201.119.29:9420/ix/api/seo/v1/footer'; // https://shop.fandom.com/ix/api/seo/v1/footer 500 - Internal Server Error
+		
+		$memcKey = wfMemcKey( self::MEMC_PREFIX_FANDOM_STORE, $wgCityId );
+
+		// do api request
+		$client = new Client( [
+			'base_uri' => $uri,
+			'timeout' => 30.0
+		] );
+		$params = [
+			'clientId' => 'fandom',
+			'relevanceKey' => $wgFandomStoreMap[ $wgCityId ],
+		];
+
+		try {
+			$response = $client->get( '', [
+				'query' => build_query( $params, PHP_QUERY_RFC1738 ),
+			] );
+			$data = json_decode( $response->getBody() );
+			// store in cache indefinitely
+			$wgMemc->set( $memcKey, $data, self::TTL_INFINITE );
+			return $data;
+		}
+		catch ( ClientException $e ) {
+			WikiaLogger::instance()->error( self::FANDOM_STORE_ERROR_MESSAGE, [
+				'error_message' => $e->getMessage(),
+				'status_code' => $e->getCode(),
+			] );
+
+			throw new WikiaException( self::FANDOM_STORE_ERROR_MESSAGE, 500, $e );
```

Comment:
Is this really an exception? How this throw will affect app flow?

I'll suggest that if we cannot read the API, we just notice the logger and silently return null;

Reply:
No longer throwing the error and returning `null`.